### PR TITLE
Added Pagination system for Comments in BO

### PR DESCRIPTION
**Pull request by Oksydan**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | I added pagination system for comments in BO. In ProductComment I removed unused param `deleted` in method getByValidate and added few more to handle pagination. Also I added new method to this class called getCountByValidate to get value for listTotal for helper list.
| Type?         | improvement
| BC breaks?    | no
| Deprecations? |  no
| Fixed ticket? | Fixes https://github.com/PrestaShop/PrestaShop/issues/15597
| How to test?  | Add few comments (copy them in sql it will be faster). Change moderate option or accept few by yourself.

### CODE REVIEW

**Comment by Oksydan** [comment]
@Progi1984 
ping

File: ProductComment.php [hunk 1]
```diff
@@ -277,19 +277,44 @@ public static function getGradedCommentNumber($id_product)
      *
      * @return array Comments
      */
-    public static function getByValidate($validate = '0', $deleted = false)
+    public static function getByValidate($validate = '0', $p = null, $limit = null, $skip_validate = false)
```
Comment (Progi1984) [inline ProductComment.php:280]: Please don't remove $deleted parameter ;)

Comment (Oksydan) [reply ProductComment.php:280]: @Progi1984 

Why? $deleted param is unused, why it should stay ?

Comment (Progi1984) [reply ProductComment.php:280]: Because, it's a breaking change to remove it ;)

Comment (Oksydan) [reply ProductComment.php]: @Progi1984 

$deleted param has been restored

Comment (Progi1984) [reply ProductComment.php]: :+1:

**Comment by Oksydan** [comment]
@matks @Progi1984 
Why this PR got closed?

**Comment by matks** [comment]
Not intended, probably done by github

**Review by matthieu-rolland — APPROVED** [review]

**Comment by sarahdib** [comment]
Hello @Oksydan 

I add 30 comments to approuve but pagination didn't display : 
https://drive.google.com/file/d/1YmPWzsfAwLF-3o5_qMCMP4osKHPeaexZ/view

**Comment by Oksydan** [comment]
@sarahdib

Pagination system has been added only to already approved comments and limit is set to 50.
screencapture-clean1765-test-admin-sklep-index-php-2020-09-19-09_18_44

**Comment by Progi1984** [comment]
PIng @Oksydan & @sarahdib

**Pull request merged**